fxa-client/src/lib.rs

```diff
@@ -496,6 +496,15 @@ impl FirefoxAccount {
         self.state.config.token_server_endpoint_url()
     }
 
+    pub fn get_connection_success_endpoint_url(&self) -> Result<Url> {
+        match self.state.config.content_url_path("connect_another_device") {
```

Comment:
Probably use `?` instead of this match. e.g. `let url = self.state.config.content_url_path("connect_another_device")?`.